aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -479,10 +479,13 @@ std::unique_ptr<TensorIterator> TensorIterator::binary_op(Tensor& out, const Ten
   return builder.build();
 }
 
-std::unique_ptr<TensorIterator> TensorIterator::unary_op(Tensor& out, const Tensor& a) {
+std::unique_ptr<TensorIterator> TensorIterator::unary_op(Tensor& out, const Tensor& a, bool resize_outputs) {
   auto builder = TensorIterator::Builder();
   builder.add_output(out);
   builder.add_input(a);
+  if (!resize_outputs) {
```

Comment:
Down the road, is this something we want to remove again? This can be part of a wider discussion around operator contracts.

Reply:
This is topic for larger discussion. I assume copy should only accept same shapes of src dst (as it was before), but if we want to have broadcasting here, it is also possible as soon as we rewrite TH code.